tools/rebuildPtreeRst.py

```diff
@@ -0,0 +1,42 @@
+import os.path
+import textwrap
+
+from pyqtgraph.parametertree.Parameter import PARAM_TYPES, _PARAM_ITEM_TYPES
+
+
+def mkDocs(typeList):
+    typDocs = [
+    f"""\
+    .. autoclass:: {typ.__module__}.{typ.__name__}
```

Comment:
I think just `typ.__name__` could be used here to achieve what I mentioned above about the signature in the docs.